packages/react/src/Paragraph/Paragraph.tsx

```diff
@@ -6,19 +6,26 @@
 
 import clsx from 'clsx'
 import { ForwardedRef, forwardRef, HTMLAttributes, PropsWithChildren } from 'react'
+import { gridCellClassNames, GridCellProps } from '../grid'
 
-export interface ParagraphProps extends HTMLAttributes<HTMLParagraphElement> {
+export type ParagraphProps = {
```

Comment:
Is there a reason to prefer `interface` over `type`? Extending _and_ intersecting doesn’t work, intersecting `HTMLAttributes` does and seems less complex?

Reply:
I do think it will prove useful to assign a combination of reusable prop objects (interfaces or type aliases) to a component like `Paragraph`.

In general, both concepts support combinations: interfaces can be [extended](https://www.typescriptlang.org/docs/handbook/interfaces.html#extending-interfaces), type aliases can be [intersected](https://www.typescriptlang.org/docs/handbook/2/objects.html#intersection-types).

The reason for my comment is that I wanted to add `& GridColumnProps` to this `ParagraphProps` interface to make a paragraph accept props like `grid-column-start`, and found that intersections aren’t for interfaces. That’s why I converted it to a type alias for now. For the Amsterdam app, we use them exclusively instead of interfaces. Made me curious after the difference and why someone chose interfaces over type aliases here.

While researching these terms I learned:
* “Differences in how conflicts are handled is the main reason to pick one over the other between an interface and a type alias of an intersection type.”
* “Type aliases, unlike interfaces, can describe more than just object types.”
* “Type aliases and interfaces are very similar, and in many cases you can choose between them freely. Almost all features of an interface are available in type, the key distinction is that a type cannot be re-opened to add new properties vs an interface which is always extendable.”

”For the most part, you can choose based on personal preference, and TypeScript will tell you if it needs something to be the other kind of declaration. If you would like a heuristic, use interface until you need to use features from type” ([source](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces)).

 We could go for interfaces here as we already started out with them and the TS docs suggest so. I’ll check if I can use them in my grid work as well.

Reply:
I’ll revert this change from `interface` to `type` for paragraph as the subject of this branch is grids. Interfaces can be extended from multiple types. Let’s continue this discussion later.